docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Conversion/MapKeyToValueMultiColumn.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using Microsoft.ML.SamplesUtils;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public class MapKeyToValueMultiColumn
+    {
+        public static void Example()
+        {
+            // Create a new context for ML.NET operations. It can be used for exception tracking and logging, 
+            // as a catalog of available operations and as the source of randomness.
+            // Setting the seed to a fixed number in this example to make outputs deterministic.
+            var mlContext = new MLContext(seed: 0);
+
+            // Create a list of data examples.
+            var examples = DatasetUtils.GenerateRandomMulticlassClassificationExamples(1000);
```

Comment:
can we have inline data like the other sample?

Reply:
I feel like this is the one use case people will need to use MapKeyToValue: results of a binary or multiclass trainer, therefore i appended to one such example. 


---
In reply to: [271977941](https://github.com/dotnet/machinelearning/pull/3187#discussion_r271977941) [](ancestors = 271977941)

Reply:
chatted with Zeeshan and Rogan too. Changing it for different values. 

---
In reply to: [272288386](https://github.com/dotnet/machinelearning/pull/3187#discussion_r272288386) [](ancestors = 272288386,271977941)

Reply:
I feel like the most common use case for this transform would be this one: after multiclass/binary get back the original values, therefore used it in this context.  

---
In reply to: [271977941](https://github.com/dotnet/machinelearning/pull/3187#discussion_r271977941) [](ancestors = 271977941)

Reply:
Gani's PR is not checked in. I can switch the Generate after he checks in. 

---
In reply to: [272332287](https://github.com/dotnet/machinelearning/pull/3187#discussion_r272332287) [](ancestors = 272332287,271977941)